hl/fortran/src/CMakeLists.txt

```diff
@@ -318,3 +318,51 @@ if (HDF5_EXPORTED_TARGETS)
       INCLUDES DESTINATION include
   )
 endif ()
+
```

Comment:
Missing fortran pkconfig files. (Cut and paste from other src folders, with appropriate substitutions)